script.js

```diff
@@ -0,0 +1,206 @@
+    pimage: './img/workimg/work2.svg',
+    ptech: ['html', 'css', 'javascript'],
+    plive: 'https://dheerajsachdeva.github.io/Portfolio-Mobile-First/',
+    psource: 'https://github.com/dheerajsachdeva/Portfolio-Mobile-First/tree/animation',
+  },
+
+  {
+    pid: '3',
+    pname: 'Tonic',
+    pnamedetails: 'Canopy',
+    pnamedetails1: 'BackEnd Dev',
+    pnamedate: '2015',
+    pdescription: 'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Beatae blanditiis, tenetur dolor rerum quod minima animi sunt molestias maiores perferendis facilis? Modi, fuga. Deleniti nihil, aut blanditiis corrupti fugit ducimus.Lorem ipsum dolor, sit amet consectetur adipisicing elit. Beatae blanditiis, tenetur dolor rerum',
+    pdescription1: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    pimage: './img/workimg/work3.svg',
+    ptech: ['html', 'css', 'javascript'],
+    plive: 'https://dheerajsachdeva.github.io/Portfolio-Mobile-First/',
+    psource: 'https://github.com/dheerajsachdeva/Portfolio-Mobile-First/tree/animation',
+  },
+
+  {
+    pid: '4',
+    pname: 'Tonic',
+    pnamedetails: 'Canopy',
+    pnamedetails1: 'BackEnd Dev',
+    pnamedate: '2015',
+    pdescription: 'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Beatae blanditiis, tenetur dolor rerum quod minima animi sunt molestias maiores perferendis facilis? Modi, fuga. Deleniti nihil, aut blanditiis corrupti fugit ducimus.Lorem ipsum dolor, sit amet consectetur adipisicing elit. Beatae blanditiis, tenetur dolor rerum',
+    pdescription1: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    pimage: './img/workimg/work4.svg',
+    ptech: ['html', 'css', 'javascript'],
+    plive: 'https://dheerajsachdeva.github.io/Portfolio-Mobile-First/',
+    psource: 'https://github.com/dheerajsachdeva/Portfolio-Mobile-First/tree/animation',
+  },
+];
+
+// FOR PAGE
+
+const cards = document.querySelector('.cards');
+
+cards.innerHTML = '';
+
+for (let i = 0; i < pdetails.length; i += 1) {
+  let tech1 = '';
+  pdetails[i].ptech.forEach((ptech) => {
+    tech1 += `<li>${ptech}</li>`;
+  });
+
+  cards.innerHTML += ` 
+  <div class='items work1'>
+  <div class='workimg'> <img      id='work1-img-mob' src='${pdetails[i].pimage}' alt='Work Image Screenshot1'>
+      <img id='work1-img-desk' src='${pdetails[i].pimage}' alt='Work Image Screenshot1'>
+
+      </div>
+      <div class='work1-text'>
+      <h1>${pdetails[i].pname}</h1>
+      <ul class='work-ul'>
+          <li>
+              <h2>${pdetails[i].pnamedetails}</h2>
+          </li>
+          <li>
+              <h3><img src='./img/workimg/Counter.svg'> &nbsp; &nbsp; ${pdetails[i].pnamedetails1}&nbsp; &nbsp; <img
+                      src='./img/workimg/Counter.svg'> &nbsp; &nbsp;${pdetails[i].pnamedate} </h3>
+          </li>
+      </ul>
+      <p> ${pdetails[i].pdescription1}</p>
+      <ul class='tech work-tags'>
+      ${tech1}
+      </ul>
```

Comment:
- [ ] Firstly, I must say that you did an awesome job using javascript to render your object data structured with HTML. However, when the pop-up is clicked, the whole card seems to be shifted to the right instead of centered. This definitely has to do with the styling of the area I highlighted. Fixing this gives a much better user experience. Put yourself in the shoes of a hiring manager, and what your reaction will be if you review such sample projects 😉 

- I've made a detailed comparison of your design and what's needed based on [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) given:

# Your desktop pop-up view:
![image](https://user-images.githubusercontent.com/50990925/216304032-46df8fdf-05f7-4d24-a645-8a991f86c2e8.png)

# Rightly centered by figma design 👍 
![image](https://user-images.githubusercontent.com/50990925/216304099-c9c10e40-735d-49b1-adc9-ce9deedb7e70.png)


- [ ] On the mobile pop-up, the buttons are completely hidden :

# Your view:
![image](https://user-images.githubusercontent.com/50990925/216304395-caf131ac-fa47-4b1e-acb6-1ee96d31f794.png)

# Righly placed on figma:
![image](https://user-images.githubusercontent.com/50990925/216304541-84a99269-4647-4427-9589-cd438f71f28a.png)